src/StrawberryfieldUtilityService.php

```diff
@@ -332,7 +284,50 @@ public function verifyDrush(string $execpath, string $home) :bool {
     return $canexecute;
   }
 
+  /**
+   * {@inheritdoc}
+   */
+  public function getCountByProcessorInSolr(EntityInterface $entity, string $processor, array $indexes = [], string $checksum = ''): int {
```

Comment:
Thinking loud here and being a little too strict, but for consistency we tend to make not required options by default NULL. If you are checking for $checksum as a string and because of how many things can be validate to nothing, I would either check for
strict $checksum !== '' or go this simpler route, where NULL can actually be omitted as an option but will cleanly (and also semantically be congruent) with the fact we are not giving any. Not a blocker. But reads in general better.

Reply:
I don't have a strong preference, I tend to put the "empty" value for the data type (array is [], string ''...) but you're right that if we have a standard, I should follow it.

Reply:
Thanks a lot!